src/partials/benefits/benefits.css

```diff
@@ -1,3 +1,100 @@
-.benefits {
-  background-color: saddlebrown;
+.benefits-wrapper {
+  padding-top: 31.5px;
+  padding-bottom: 20.5px;
 }
+
+.benefits-section-title {
+  font-family: "Inter", sans-serif;
```

Comment:
If you leve this class plaese change font-family to font-family: 'Inter Tight', sans-serif;

Reply:
I don't want to change this class, because ".section-title"  differ
![image](https://github.com/gudzsv/project-syntax-samurais-02/assets/150190335/37d6b1df-bffa-4596-8547-2c39af282f4f)